warp/tests/test_static.py

```
@@ -689,6 +773,12 @@ def test_static_python_call(self):
 add_function_test(TestStatic, "test_static_constant_hash", test_static_constant_hash, devices=None)
 add_function_test(TestStatic, "test_static_function_hash", test_static_function_hash, devices=None)
 add_function_test(TestStatic, "test_static_len_query", test_static_len_query, devices=None)
+add_function_test(
```

Ruff (0.14.10)

`add_function_test` may be undefined, or defined from star imports

(F405)